src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,10 @@
+
+
+export const ArticleCard = ({article}) => {
+    return (
+        <section className="article-card">
+<h3>{article.title}</h3>
+<img src={article.article_img_url} alt="article image" />
+        </section>
```

Comment:
Great job showing the article's title - but there's quite a bit more data on an article than just it's title. Which of these would be useful for a user to see, and can we show them? :)